docs/best_practices/tool_xml.rst

```diff
@@ -135,6 +135,9 @@ escaping (``&amp;``).
 
 `Wikipedia has more on CDATA <http://en.wikipedia.org/wiki/CDATA>`__
 
+All Cheetah variables for text parameters, input and output files must be
+single-quoted, e.g. ``'${var_name}'``.
```

Comment:
I thought mostly we used a single double-quote character for filenames, i.e. ``"${var_name}"``.